types/lodash/common/lang.d.ts

```diff
@@ -1097,13 +1097,13 @@ declare module "../index" {
          * _.isObjectLike(null);
          * // => false
          */
-        isObjectLike(value?: any): boolean;
```

Comment:
This doesn't look correct: type `object` includes functions in TypeScript, but `isObjectLike` returns `false` for functions. For this to work, you'd have to use a type that expresses all non-function objects. Maybe using `Record`?

Reply:
On second thought, I don't think TS can actually express the type of non-function objects at the moment, actually.